lib/govuk_index/publishing_event_worker.rb

```diff
@@ -25,6 +26,8 @@ def perform(routing_key, payload)
     # the process to continue
     rescue NotFoundError
       Services.statsd_client.increment('govuk_index.not-found-error')
+    rescue UnknownDocumentTypeError
+      Services.statsd_client.increment('govuk_index.unknown-document-type')
```

Comment:
I'm not sure on this, but it might be helpful when we are deciding what to do next:
```
rescue UnknownDocumentTypeError
  Services.statsd_client.increment("govuk_index.unknown-document-type.#{e.message}")
```

where the message is the document_type from the payload. 〰️